mm2src/coins/lp_coins.rs

```diff
@@ -3194,6 +3194,10 @@ pub enum WithdrawError {
     IBCChannelNotHealthy {
         channel_id: ChannelId,
     },
+    #[display(fmt = "IBC channel '{}' is not present on the target node.", channel_id)]
+    IBCChannelMissingOnNode {
+        channel_id: ChannelId,
+    },
```

Comment:
Shouldn't we combine all these errors as `IBCChannelError`?

Reply:
Good point, I will try that.